WriterDesk1/backend/tests/conftest.py

```diff
@@ -129,6 +129,12 @@ def initDatabaseEmpty(testClient):
     # Create the database:
     db.create_all()
 
+    admin = User('ad', 'min')
+    db.session.add(admin)
+    admin.role = 'admin'
+
+    db.session.commit()
+    
```

Comment:
dit staat al in initDatabase, gebruik die

er is een reden dat ie initDatabase**Empty** heet XD

Reply:
Maar elke test die een lege database heeft moet minstens 1 admin hebben om API calls te maken met JWT required;
Oke als ik deze laat staan?

Reply:
good point, is goed zo dan